.travis.yml

```diff
@@ -2,24 +2,31 @@ language: cpp
 os: linux
 compiler: gcc
 
+cache:
+- directories:
+  - $HOME/.cache
+
 addons:
   apt:
     packages: lcov
 
-before_script:
-- mkdir -p build && cd build
+install:
+# (fake) install dependencies
```

Comment:
My idea was that there is a concise explanation how to do code coverage and a starting example that can be used for own projects and only needs to be adjusted. The former is achieved by the `README.MD` which summarizes the parts required for code coverage. Those can also be found in the scripts. But the scripts **additionally** provide resources on how to go further. Especially important is the `lcov --remove` part in combination with tests and the cache as the pattern is not obvious.

Reply:
Going for the example to be a basis to start from is nice, albeit challenging.

Looking over the whole, I'm trying to see how hard it is to pick out just what is needed to get coverage on an existing project.
For me, your remark on [keeping it look as simple at it is *red.*](https://github.com/codecov/example-cpp11-cmake/pull/11#issuecomment-452679941) helped me to get the focus on the important parts. I'd like to return the advice.
It looks good to me. I appreciate the commenting around some basic operations.

Reply:
> Looking over the whole, I'm trying to see how hard it is to pick out just what is needed to get coverage on an existing project.

That's where the Readme comes into play. If you "just want to add coverage" the readme has these information. If you want to start a new project with coverage you take the example directly.

The code should contain all parts required for a project: src, tests, 3rd party deps (usually cached) and the build script to handle errors. What I want to avoid is users thinking "But I have <XXX>. What shall I do now?" by providing a solution to the common build setup alongside the documentation for the minimum changes

Anything which is to complicated and should be simplified?

Reply:
As the old Chinese saying says: "code speaks more than any documentation".

Looks good to me.